### ISSUE TITLE
Game Crashes

### ISSUE THREAD

**Issue by username227** [flathub/org.yuzu_emu.yuzu#292]
The game Dragon Quest Builders crashes yuzu immediately upon loading. I'm attaching a log as requested.
[yuzu_log.txt](https://github.com/flathub/org.yuzu_emu.yuzu/files/7906347/yuzu_log.txt)

**Comment by liushuyu** [comment]
Can you try the app image version and see whether the issue persists? https://github.com/yuzu-emu/yuzu-mainline/releases/download/mainline-0-906/yuzu-20220202-17d378353.AppImage

That way I would know where to forward your issue because it seems like this might be beyond my control.

**Comment by username227** [comment]
Yeah, it is still the same; using a different distro, though; Linux Mint 20.3 with latest flatpak version. crashes after starting the game.

**Comment by liushuyu** [comment]
> Yeah, it is still the same; using a different distro, though; Linux Mint 20.3 with latest flatpak version. crashes after starting the game.

Could you capture another log? Also could you also attach a copy of the `inxi -v3` output?

**Comment by username227** [comment]
[yuzu_log.txt](https://github.com/flathub/org.yuzu_emu.yuzu/files/8121446/yuzu_log.txt)
This is the log from the latest yuzu flatpak as of today, 2/22/22. Thanks for all your help on this.
This log is for Dragon Quest Builders.

As a side point, I'd also like to report what appears to be a known issue with Dragon Quest Builders 2, and that is about halfway through the opening tutorial, the graphics glitch; yuzu doesn't crash per se, but the graphics go all haywire, and it kind of freezes up. I could possibly get a log for this as well if you believe that the log would capture the problem here. Thanks!

**Comment by liushuyu** [comment]
Seems to be related to this issue: https://github.com/yuzu-emu/yuzu/issues/2052

**Comment by liushuyu** [comment]
> [yuzu_log.txt](https://github.com/flathub/org.yuzu_emu.yuzu/files/8121446/yuzu_log.txt) This is the log from the latest yuzu flatpak as of today, 2/22/22. Thanks for all your help on this. This log is for Dragon Quest Builders.
> 
> As a side point, I'd also like to report what appears to be a known issue with Dragon Quest Builders 2, and that is about halfway through the opening tutorial, the graphics glitch; yuzu doesn't crash per se, but the graphics go all haywire, and it kind of freezes up. I could possibly get a log for this as well if you believe that the log would capture the problem here. Thanks!

According to one of our team members, can you try turning off "Use asynchronous GPU emulation" option in the "Graphics" settings for this game?

**Comment by username227** [comment]
> According to one of our team members, can you try turning off "Use asynchronous GPU emulation" option in the "Graphics" settings for this game?
Based on what you were quoting, I assume you meant this for the DQB2 side issue i mentioned. Unfortunately, it didn't make a difference.

And yes, it would seem that my issue might be related to that issue you mentioned. If it's a duplication I apologize. Hopefully a fix can be found soon!

**Comment by username227** [comment]
interestingly, on the Windows version of Yuzu, Dragon Quest Builders (1) does not crash Yuzu. The game is still not playable (the menus are not displaying correctly), but i thought this was interesting.
